mautic/mautic
# Support lock for mautic:custom-field:create-column command

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Add lock for m`autic:custom-field:create-column`



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create contact field and run `mautic:custom-field:create-column command`  on the large instance
3. Run meanwhile `mautic:custom-field:create-column command`  again
4. You should see lock warning `Script in progress. Can force execution by using --bypass-locking.`
